SpectREM/SpectREM/Emulation Core/ZX_Spectrum_48k/ZXSpectrum48.cpp

```diff
@@ -320,7 +320,7 @@ void ZXSpectrum48::resetMachine(bool hard)
 
 void ZXSpectrum48::resetToSnapLoad()
 {
-    loadROM( cSMART_ROM, 0 );
+    loadROM(cSMART_ROM, 0 );
```

Comment:
I see the space, but I checked the diff for these files and there a load of them that I never did, maybe it's VS formatting the file?
If I reset these files to the originals on your repo then I would lose any changes I actually did make wouldn't I?